artemis-core-client/src/test/java/org/apache/activemq/artemis/api/core/management/AddressSettingsInfoTest.java

```diff
@@ -30,65 +30,67 @@ public class AddressSettingsInfoTest {
-         "\"autoDeleteAddressesDelay\":3003,\n" +
-         "\"redeliveryCollisionAvoidanceFactor\":1.1,\n" +
-         "\"retroactiveMessageCount\":101,\n" +
-         "\"autoCreateDeadLetterResources\":true,\n" +
-         "\"deadLetterQueuePrefix\":\"FOO.\",\n" +
-         "\"deadLetterQueueSuffix\":\".FOO\",\n" +
-         "\"autoCreateExpiryResources\":true,\n" +
-         "\"expiryQueuePrefix\":\"BAR.\",\n" +
-         "\"expiryQueueSuffix\":\".BAR\",\n" +
-         "\"expiryDelay\":404,\n" +
-         "\"minExpiryDelay\":40,\n" +
-         "\"maxExpiryDelay\":4004,\n" +
-         "\"enableMetrics\":false\n" +
-         "}";
+      String json = """
+         {
+         "addressFullMessagePolicy":"fullPolicy",
+         "maxSizeBytes":500,
+         "pageSizeBytes":200,
+         "pageCacheMaxSize":3,
+         "maxDeliveryAttempts":3,
+         "redeliveryDelay":70000,
+         "redeliveryMultiplier":1.5,
+         "maxRedeliveryDelay":100000,
+         "deadLetterAddress":"deadLettersGoHere",
+         "expiryAddress":"",
+         "defaultLastValueQueue":true,
+         "redistributionDelay":10004,
+         "sendToDLAOnNoRoute":true,
+         "slowConsumerThreshold":200,
+         "slowConsumerCheckPeriod":300,
+         "slowConsumerPolicy":"retire",
+         "autoCreateJmsQueues":true,
+         "autoDeleteJmsQueues":false,
+         "autoCreateJmsTopics":true,
+         "autoDeleteJmsTopics":false,
+         "autoCreateQueues":false,
+         "autoDeleteQueues":false,
+         "autoCreateAddresses":false,
+         "autoDeleteAddresses":false,
+         "configDeleteQueues":"OFF",
+         "configDeleteAddresses":"FORCE",
+         "maxSizeBytesRejectThreshold":1023,
+         "defaultLastValueKey":"yyy",
+         "defaultNonDestructive":false,
+         "defaultExclusiveQueue":false,
+         "defaultGroupRebalance":false,
+         "defaultGroupBuckets":1026,
+         "defaultGroupFirstKey":"xxx",
+         "defaultMaxConsumers":1001,
+         "defaultPurgeOnNoConsumers":false,
+         "defaultConsumersBeforeDispatch":1005,
+         "defaultDelayBeforeDispatch":1003,
+         "defaultQueueRoutingType":"MULTICAST",
+         "defaultAddressRoutingType":"ANYCAST",
+         "defaultConsumerWindowSize":2001,
+         "defaultRingSize":999,
+         "autoDeleteCreatedQueues":false,
+         "autoDeleteQueuesDelay":4,
+         "autoDeleteQueuesMessageCount":8,
+         "autoDeleteAddressesDelay":3003,
+         "redeliveryCollisionAvoidanceFactor":1.1,
+         "retroactiveMessageCount":101,
+         "autoCreateDeadLetterResources":true,
+         "deadLetterQueuePrefix":"FOO.",
+         "deadLetterQueueSuffix":".FOO",
+         "autoCreateExpiryResources":true,
+         "expiryQueuePrefix":"BAR.",
+         "expiryQueueSuffix":".BAR",
+         "expiryDelay":404,
+         "minExpiryDelay":40,
+         "maxExpiryDelay":4004,
+         "enableMetrics":false
+         }
+         """;
```

Comment:
Not actually an issue, especially for the usage here, but just noting for awareness...the text-block will subtly differ from the original string concats in having a newline at the end, since the closing delimiter is on its own line rather than the same line [and the block also doesnt use the trailing \\<line-terminator\> escape to suppress a line ending being added when the block is processed].